osu.Game.Tests/Visual/Multiplayer/TestSceneMultiplayerMatchSubScreen.cs

```diff
@@ -286,6 +286,27 @@ public void TestNextPlaylistItemSelectedAfterCompletion()
             });
         }
 
+        [Test]
+        [FlakyTest] // See above
```

Comment:
Please explain?

Reply:
First time contributer here, I heavily oriented myself on the existing tests in the class, all of which had this header. 
AFAIK the `TestCreatedRoom` has a small chance to error out and since the other tests rely on it, this flag is set. But I may be mistaking.

Reply:
Okay, I just wanted to make sure it wasn't related to the new changes specifically.